src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/ArtifactsSizes/centos.8-x64.txt

```diff
@@ -3124,462 +3124,462 @@
+Microsoft.Build.Utilities.Core.x.y.z-23418-03.nupkg: 696042
+Microsoft.Build.x.y.z-23418-03.nupkg: 3412956
+Microsoft.CodeAnalysis.Analyzers.x.y.z-beta1.final.nupkg: 1285173
+Microsoft.CodeAnalysis.Analyzers.x.y.z-betax.y.z.nupkg: 1285179
 Microsoft.CodeAnalysis.Analyzers.x.y.z.nupkg: 1285156
-Microsoft.CodeAnalysis.AnalyzerUtilities.x.y.z: 288749
-Microsoft.CodeAnalysis.AnalyzerUtilities.x.y.z: 288743
+Microsoft.CodeAnalysis.AnalyzerUtilities.x.y.z-beta1.final.nupkg: 288743
+Microsoft.CodeAnalysis.AnalyzerUtilities.x.y.z-betax.y.z.nupkg: 288749
 Microsoft.CodeAnalysis.AnalyzerUtilities.x.y.z.nupkg: 288724
-Microsoft.CodeAnalysis.Build.Tasks.x.y.z: 264050
-Microsoft.CodeAnalysis.Common.x.y.z: 7401611
-Microsoft.CodeAnalysis.CSharp.CodeStyle.x.y.z: 1682095
-Microsoft.CodeAnalysis.CSharp.Features.x.y.z: 4801305
-Microsoft.CodeAnalysis.CSharp.Scripting.x.y.z: 137994
-Microsoft.CodeAnalysis.CSharp.Workspaces.x.y.z: 2070666
-Microsoft.CodeAnalysis.CSharp.x.y.z: 17993577
-Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.x.y.z: 65739
-Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler.x.y.z: 24586
-Microsoft.CodeAnalysis.Features.x.y.z: 10069377
-Microsoft.CodeAnalysis.NetAnalyzers.x.y.z: 4434325
-Microsoft.CodeAnalysis.NetAnalyzers.x.y.z: 4434318
+Microsoft.CodeAnalysis.Build.Tasks.x.y.z-x.y.z.nupkg: 264050
+Microsoft.CodeAnalysis.Common.x.y.z-x.y.z.nupkg: 7401611
+Microsoft.CodeAnalysis.CSharp.CodeStyle.x.y.z-x.y.z.nupkg: 1682095
+Microsoft.CodeAnalysis.CSharp.Features.x.y.z-x.y.z.nupkg: 4801305
+Microsoft.CodeAnalysis.CSharp.Scripting.x.y.z-x.y.z.nupkg: 137994
+Microsoft.CodeAnalysis.CSharp.Workspaces.x.y.z-x.y.z.nupkg: 2070666
+Microsoft.CodeAnalysis.CSharp.x.y.z-x.y.z.nupkg: 17993577
+Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.x.y.z-x.y.z.nupkg: 65739
+Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler.x.y.z-x.y.z.nupkg: 24586
+Microsoft.CodeAnalysis.Features.x.y.z-x.y.z.nupkg: 10069377
+Microsoft.CodeAnalysis.NetAnalyzers.x.y.z-1.nupkg: 4434325
+Microsoft.CodeAnalysis.NetAnalyzers.x.y.z.final.nupkg: 4434318
 Microsoft.CodeAnalysis.NetAnalyzers.x.y.z.nupkg: 4434296
-Microsoft.CodeAnalysis.Razor.Tooling.Internal.x.y.z: 532057
-Microsoft.CodeAnalysis.Scripting.Common.x.y.z: 464236
-Microsoft.CodeAnalysis.VisualBasic.CodeStyle.x.y.z: 1316414
-Microsoft.CodeAnalysis.VisualBasic.Features.x.y.z: 3115504
-Microsoft.CodeAnalysis.VisualBasic.Workspaces.x.y.z: 1848400
-Microsoft.CodeAnalysis.VisualBasic.x.y.z: 14538374
-Microsoft.CodeAnalysis.Workspaces.Common.x.y.z: 8301803
-Microsoft.CodeAnalysis.Workspaces.MSBuild.x.y.z: 183216
-Microsoft.CodeAnalysis.x.y.z: 9724
-Microsoft.Css.Parser.x.y.z: 52961
-Microsoft.Deployment.DotNet.Releases.x.y.z: 108986
-Microsoft.Diagnostics.NETCore.Client.x.y.z: 65008
-Microsoft.DiaSymReader.x.y.z: 124712
-Microsoft.DiaSymReader.x.y.z: 124706
+Microsoft.CodeAnalysis.Razor.Tooling.Internal.x.y.z.nupkg: 532057
+Microsoft.CodeAnalysis.Scripting.Common.x.y.z-x.y.z.nupkg: 464236
```

Comment:
This should simply be x.y.z.  The reason being once the package ships a stable version the `-` suffix will be dropped.  With the current pattern, this would cause a diff.